src/containers/AppHeader/index.tsx

```diff
@@ -39,74 +40,76 @@ interface AppHeaderProps {
     setError: ActionCreator<SetErrorAction>;
 }
 
-class AppHeader extends React.Component<AppHeaderProps> {
-    public render(): JSX.Element {
-        const {
-            simulariumFile,
-            isBuffering,
-            changeToLocalSimulariumFile: loadLocalFile,
-            changeToNetworkedFile: loadNetworkFile,
-            setViewerStatus,
-            clearSimulariumFile,
-            setError,
-            isNetworkedFile,
-        } = this.props;
-        let lastModified = 0;
-        let displayName = "";
-        if (isLocalFileInterface(simulariumFile)) {
-            displayName = simulariumFile.name;
-            lastModified = simulariumFile.lastModified;
-        } else if (isNetworkSimFileInterface(simulariumFile)) {
-            displayName = simulariumFile.title;
-        }
+const AppHeader: React.FC<AppHeaderProps> = ({
+    simulariumFile,
+    isBuffering,
+    changeToLocalSimulariumFile: loadLocalFile,
+    changeToNetworkedFile: loadNetworkFile,
+    setViewerStatus,
+    clearSimulariumFile,
+    setError,
+    isNetworkedFile,
+}) => {
+    const history = useHistory();
```

Comment:
Don't believe the diff, this functional component is practically identical to the previous class based component. except for changing the `Link` to a `NavButton` that implements `useHistory` and removing the action button container.